core/src/test/java/org/infinispan/xsite/backupfailure/tx/BaseBackupTxFailureTest.java

```diff
@@ -33,11 +32,10 @@ protected void createSites() {
       backup("LON").getAdvancedCache().getAsyncInterceptorChain().addInterceptor(failureInterceptor, 1);
    }
 
-   @Test(groups = {"xsite", "unstable"})
    public void testPrepareFailure() {
       failureInterceptor.enable();
       try {
-         Exceptions.expectException(XAException.class, () -> cache("LON", 0).put("k", "v"));
+         Exceptions.expectException(CacheException.class, () -> cache("LON", 0).put("k", "v"));
```

Comment:
You could use Exceptions.expectException(CacheException.class, RollbackException.class, () -> cache("LON", 0).put("k", "v"));
I noticed the XAException is now a suppressed exception inside the RollbackException, but exploring the whole exception hierarchy might be overkill